api/v1beta1/openstackclient_types.go

```diff
@@ -35,6 +35,9 @@ type OpenStackClientSpec struct {
 	// +kubebuilder:default={ctlplane,external}
 	// Networks the name(s) of the OpenStackNetworks used to generate IPs
 	Networks []string `json:"networks"`
+
+	// StorageClass to be used for the controller disks
```

Comment:
Maybe just to avoid confusion, this should say "client" instead of "controller"?  Otherwise this PR looks good

Reply:
done